src/Modules/ProjectModule/components/ProjectList/ProjectList.tsx

```diff
@@ -77,13 +116,27 @@ export default function ProjectList() {
                                       View
                                     </span>
                                     <span className="text-success">
-                                      <i className="fa-solid fa-pen-to-square my-2"></i>{" "}
-                                      Edit
+                                      <Link
+                                        to={`/dashboard/project-data/${project.id}`}
+                                        className="text-decoration-none"
+                                        state={{
+                                          updateData: project,
+                                          type: "update",
+                                        }}
+                                      >
+                                        <i className="fa-solid fa-pen-to-square text-warning my-1"></i>{" "}
+                                        Edit
+                                      </Link>{" "}
                                     </span>
                                     <span className="text-success">
-                                      {" "}
-                                      <i className="fa-solid fa-trash my-1"></i>{" "}
-                                      Delete
+                                      <div
+                                        onClick={() =>
```

Comment:
We shouldn't be giving a div an onClick since it is not an interactive element. We need to replace this with a button.